stripe/src/main/java/com/stripe/android/IssuingCardPinService.kt

```diff
@@ -8,15 +8,20 @@ import com.stripe.android.Stripe.Companion.appInfo
 import com.stripe.android.exception.InvalidRequestException
 import com.stripe.android.networking.StripeApiRepository
 import com.stripe.android.networking.StripeRepository
-import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.CoroutineScope
```

Comment:
I am curious how you found this?   DO we have checks in place to make sure it doesn't happen other places?

Reply:
did a quick search, it looks like apart from the synchronous calls [here](https://github.com/stripe/stripe-android/blob/2202bdd139c3b07b8be685003b7c43f9289d3a6e/stripe/src/main/java/com/stripe/android/Stripe.kt#L44), there is no prod code using runBlocking